packages/react-components/react-slider/src/stories/Slider/SliderControlled.stories.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Label } from '@fluentui/react-label';
 import { useId } from '@fluentui/react-utilities';
 import { Button } from '@fluentui/react-button';
-import { Slider, SliderProps } from '../../../index';
+import { Slider, SliderProps } from '@fluentui/react-slider';
```

Comment:
_nit:_ You could split these up into

```suggestion
import { Slider } from '@fluentui/react-slider';
import type { SliderProps } from '@fluentui/react-slider';
```